observability/readme.md

```diff
@@ -17,14 +17,14 @@ Building observable systems enables development teams at CSE to measure how well
 
 ## Recommended Practices
 
+1. **Correlation Id**: Include unique identifier at the start of the interaction to tie down aggregated data from various system components and provide a holistic view. Read more guidelines about using [correlation id](correlation-id.md).
 1. Ensure health of the services are **monitored** and provide insights into system's performance and behavior.
```

Comment:
This numbering should probably be 1. 2. ... instead of 1. 1. 2. 

Suggestion: run the markdownlint extension in VS Code to catch style issues